Scripts/Invoke-JiraApiRequest.ps1

```diff
@@ -0,0 +1,31 @@
+[Diagnostics.CodeAnalysis.SuppressMessage(
+    'PSAvoidUsingConvertToSecureStringWithPlainText',
+    '',
+    # Also see: https://github.com/PowerShell/PSScriptAnalyzer/issues/574.
+    Justification = 'The API token already comes from a secure store. Under GHA, we can''t make it any more secure.')]
+param (
+    [string]$ApiEndpoint,
```

Comment:
If indeed this cannot be anything than `https://<yourdomain>.atlassian.net`, then we can validate against that.

Reply:
I just learned that it can be a custom domain too: https://support.atlassian.com/organization-administration/docs/add-a-custom-domain/

Reply:
Gotcha!